### ISSUE TITLE
How to change project dir()

### ISSUE THREAD

**Issue by Jayasurya-Vardhan** [OpenEnergyPlatform/open-MaStR#394]
Hello everyone, 

for the past two months, I've been working on the Open-MaStR tool. It is being used for our project, Wasserstoffatlas (https://www.wasserstoffatlas.de/). This is a really helpful package for getting MaStR data, starting from the download of an XML file to exporting it as a CSV.

However, we have recently encountered a problem accessing the final CSV file [ bnetza_mastr_XXX_raw] from the home directory when given a path. Is it possible to change the project directory to CWD rather than the home directory so that we can access the file more easily?

We are setting everything up in such a way that it is implemented as an automated process for generic use on multiple devices, so if we have our file in CWD then it will be simpler for this to work.

Please help me with this issue @Ludee @FlorianK13 @deniztepe @chrwm 

Thank you

**Comment by nesnoj** [comment]
Hey @Jayasurya-Vardhan!

If I'm not mistaken, a different dir is currently not supported. The only easy option may be to change the source [here](https://github.com/OpenEnergyPlatform/open-MaStR/blob/88e58315f75aeaa9a2bb6a10e3ec795e0f43c0ab/open_mastr/utils/config.py#L47) to the CWD (or whatever) and a subsequent manual install (clone -> modification -> install).

Indeed, it'd be helpful to have such a customization option, e.g. an env variable which specifies the home dir.

**Comment by chrwm** [comment]
Thanks for the issue. As general idea, I suggest to introduce a paramter e.g. `directory` in the Mastr class, so that people can specify two parameters `db = Mastr(engine='sqlite', directory='cwd')` when they instantiate the class. The parameter would be passed to `get_project_home_dir(directory)` where the negotiation between options happens e.g. 

meta-code
```
def get_project_home_dir(directory: str = None):
    """Get root dir of project data

    On linux this path equals `$HOME/open-MaStR/`, respectively `~/open-MaStR/`
    which is also called `PROJECTHOME`.

    Returns
    -------
    path-like object
        Absolute path to root dir of open-MaStR project home
    """

    if directory == 'cwd':
        return os.getcwd()
    if not directory:
        return os.path.join(os.path.expanduser("~"), ".open-MaStR")

    raise NotImplementedError("Please specify a valid directory. The parameter directory is implemented for: 'cwd', None")
```
Further details in the code will be checked and methods renamed and documened accordingly.

@FlorianK13 any concerns or other ideas? If not I'd proceed with implementation.

**Comment by deniztepe** [comment]
I find such a solution feasible. What would speak against to accept any given path to be the home directory (additional to cwd option)?

**Comment by chrwm** [comment]
Thanks for the feedback, nothing I can implement it as third option.

**Comment by FlorianK13** [comment]
Could directory also be a string to any other directory or a path object? 
Then we might rename the variable 'directory' to 'path' or 'dir_path'. If it's none (default) we use the .open-mastr directory?

**Comment by Jayasurya-Vardhan** [comment]
Thank you everyone for the quick response, it would be really helpful if we have the option to customize the path.

**Comment by chrwm** [comment]
I'll implement and then we can further discuss in the PR

**Comment by Jayasurya-Vardhan** [comment]
Hello, @chrwm  we were expecting this feature in the latest release but unfortunately, it was not included in it and I apologize if something wrong from my end because I am new to GitHub so I don’t exactly know how the procedure works after raising an issue.

**Comment by FlorianK13** [comment]
Hi @Jayasurya-Vardhan the feature was not ready when we started the release procedure for 0.13.0 However it should be part of the next release if everything goes on as planned.

**Comment by FlorianK13** [comment]
@chrwm This is merged to develop, right? So it will be part of release v0.14.0 and we can close the issue?

**Comment by chrwm** [comment]
Released in v0.14.0
